runtime/src/iree/hal/drivers/hip/hip_multi_queue_command_buffer.h

```diff
@@ -0,0 +1,56 @@
+// Copyright 2024 The IREE Authors
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#ifndef IREE_HAL_DRIVERS_HIP_MULTI_QUEUE_COMMAND_BUFFER_H__
```

Comment:
Same comment in one of the other files but raising to a "fix everywhere" - one trailing _.
```suggestion
#ifndef IREE_HAL_DRIVERS_HIP_MULTI_QUEUE_COMMAND_BUFFER_H_
```